### ISSUE TITLE
Inquiry on likelihood to revive this project

### ISSUE THREAD

**Issue by lextm** [brian2github/FailedRequestTraceViewer#1]
As you might know, due to the death of IE there isn't any web browser today by default supports viewing FRT log files (as they don't load  XSL and XML together). So, a tool like this that works specific for FRT becomes important again.

If you do plan to implement the missing features in the future, then I can send you a few pull requests to address some of the issues I met and fixed.

If you don't have any plan to revive it, then I will move ahead in my own fork.

Look forward to your response.

**Comment by brianismeta** [comment]
Hi Lextm, I’d like to update the project, and get it to run again in the modern post-IE environment.  I haven’t updated it because it doesn’t seem anyone was using it.  I still use it from time-to-time but I know some things no longer work as expected, like the viewing.

**Comment by brian2github** [comment]
I made some more updates, but if you like to create a pull request I can figure out how to merge your pull request, and then check in my changes.  Otherwise I will just update the project with our combined changes.  Let me know today (in the next 24h if possible) as I want to create a new release and close the book for now.  I provided an 'non-IE' option by default which applies the xsl to the xml and generates the html.  Its OK inside the webviewer control because I can suppress errors, but if opened externally by Edge, will see that there are some script errors in the HTML it generates,  The XSL has commands to check whether certain functions are supported, and if not, hides a tab bar.   The C# transform doesn't support it (I guess) so it strips that part out, but then the are script errors because the tab bar is missing.  Probably the XSL needs to be "corrected" in-flight to avoid script errors when opening in modern browser.
            <xsl:choose>
               <xsl:when test="function-available('jsext:datediff')">

**Comment by lextm** [comment]
Created #2 . The other key changes I can think of are,

* Convert to .NET 6 (or 7)
* Add an MSI installer (WiX based)

But clearly they can wait. Let's try to release often as soon as some useful changes are there.

**Comment by brian2github** [comment]
Excellent to hear.  Will figure out how to pull in the changes when I get a chance, and put my changes on top.

**Comment by brian2github** [comment]
I don't think it needs an installer.  I think of it as a tool like the sysinternal tools (process explorer, process monitor, etc.) which individually don't have install programs.  If necessary, there is a way to pack the managed dependent assemblies into one bloated .exe (I forget offhand how) so can have a single .exe to run, or make assumptions on the expected framework that should be commonly available.  Or build it for two frameworks for release.

**Comment by lextm** [comment]
If you don't like an installer, then .NET 6 does allow to generate a single .exe file.

**Comment by brian2github** [comment]
OK after a few confusing attempts I finally got the repo updated with the pull request and my updates.  IE's view still much better, would be nice if could mimic the same capability in the HTML output, Edge/Chrome only see one view of the data, and clicking links often doesn't do anything, just have to scroll through for now.  I think fixing the XSL to give Edge/Chrome a better experience is probably the biggest value-add that I can think of right now.  Not sure if webview wpf control will switch to Edge on some machines, otherwise if webview fails due to missing IE, I think we could do without the webview as long as opening the row in the external browser (double-click) gives a good experience.  I don't want to undertake moving to edge webview2 although you're welcome to.
